Submitted By:            Randy McMurchy <randy_at_linuxfromscratch_dot_org>
Date:                    2007-04-09
Initial Package Version: 2.3.19
Upstream Status:         In upstream CVS
Origin:                  Upstream
Description:             Fixes an error during the installation of KDE-Libs


diff -Naur libart_lgpl-2.3.19.orig/art_misc.h libart_lgpl-2.3.19/art_misc.h
--- libart_lgpl-2.3.19.orig/art_misc.h  2007-01-01 18:59:22.000000000 -0500
+++ libart_lgpl-2.3.19/art_misc.h       2007-03-20 23:04:52.000000000 -0400
@@ -34,9 +34,15 @@
 #include <libart_lgpl/art_config.h>
 #endif

+#ifdef __cplusplus
+extern "C" {
+#endif
 void *art_alloc(size_t size);
 void art_free(void *ptr);
 void *art_realloc(void *ptr, size_t size);
+#ifdef __cplusplus
+}
+#endif /* __cplusplus */

 /* These aren't, strictly speaking, configuration macros, but they're
    damn handy to have around, and may be worth playing with for

